plugin/pkg/admission/limitranger/interfaces.go

```diff
@@ -22,6 +22,7 @@ import (
 	"k8s.io/apiserver/pkg/admission"
 )
 
+// LimitRangerActions is an interface defining actions to be carried over ranges to identify and manipulate their limits
 type LimitRangerActions interface {
```

Comment:
golint recommends renaming this interface to `Actions` to prevent stutter. This is the last violation remaining in the package and seems reasonable, but would break the interface for consumers. A quick search of users inside the repo shows only 2 uses:
```
vendor/github.com/kubernetes/repo-infra/kazel/kazel.go
613:	options := []string{"limitranger.LimitRangerActions", "BUILD"}

vendor/github.com/bazelbuild/bazel-gazelle/internal/config/config.go
93:var DefaultValidBuildFileNames = []string{"limitranger.LimitRangerActions", "BUILD"}
```

Reply:
That could work. I haven't addressed this yet in case there are other opinions. I'll tackle that once tests are green again.

Reply:
It's done.

Reply:
> This is the last violation remaining in the package and seems reasonable, but would break the interface for consumers

renaming the package also breaks the interface